model/Software/Classes/PackageGitoid.md

```diff
@@ -0,0 +1,23 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# PackageGitoid
+
+## Summary
+
+Preferred verification method to be used for a Package which represents an artifact that logically binds a number of single files together.
+
+## Description
+
+https://www.iana.org/assignments/uri-schemes/prov/gitoid Gitoid stands for [Git Object ID](https://git-scm.com/book/en/v2/Git-Internals-Git-Objects) and a gitoid of type blob is a unique hash of a binary artifact. 
```

Comment:
This sentence appears twice. Can we drop redundant descriptions?

Reply:
I don't see where the sentence appears twice - can you reference the other sentence?

Reply:
https://github.com/spdx/spdx-3-model/pull/498/files#diff-0d379f36e4e378c06fa5c274a23c3c113fbf6190b0ca6bb62a4b21782a54aef0R11

Reply:
Since the second reference is in the property name description, I think the redundancy is fine.